### PR TITLE
Judicial System: Court Date Notifications

### PR DESCRIPTION
**Pull request by gudjong**
# Notifications

https://app.asana.com/0/1197628843274291/1198949971788367

## What

☝️ 

## Why

Requirements

## Screenshots / Gifs

Attach Screenshots / Gifs to help reviewers understand the scope of the pull request

## Checklist:

- [ ] I have performed a self-review of my own code
- [ ] I have commented my code, particularly in hard-to-understand areas
- [ ] I have made corresponding changes to the documentation
- [ ] My changes generate no new warnings
- [ ] I have added tests that prove my fix is effective or that my feature works
- [ ] Formatting passes locally with my changes
- [ ] I have rebased against master before asking for a review

### CODE REVIEW

File: apps/judicial-system/backend/src/app/modules/notification/notification.controller.spec.ts [hunk 1]
```diff
@@ -0,0 +1,22 @@
+// import { Test, TestingModule } from '@nestjs/testing'
```
Comment (oddsson) [inline apps/judicial-system/backend/src/app/modules/notification/notification.controller.spec.ts:1]: Did you forget to implement this? Can we please do so or remove it before going forward?

File: apps/judicial-system/backend/src/app/modules/notification/notification.service.spec.ts [hunk 2]
```diff
@@ -0,0 +1,22 @@
+// import { Test, TestingModule } from '@nestjs/testing'
```
Comment (oddsson) [inline apps/judicial-system/backend/src/app/modules/notification/notification.service.spec.ts:1]: Did you forget to implement this? Can we please do so or remove it before going forward?

**Review by oddsson — CHANGES_REQUESTED** [review]
There are two test files that are not implemented.. Please consider removing them or implementing them :) Otherwise it looks really good 💪

File: apps/judicial-system/backend/src/app/modules/notification/notification.controller.spec.ts (hunk 1 above)
Comment (gudjong) [reply apps/judicial-system/backend/src/app/modules/notification/notification.controller.spec.ts:1]: I intentionally left them out and was planning to implement later. But I will remove them for now and add again when ready.

**Review by oddsson — APPROVED** [review]

**Pull request merged**